packages/loader/container-loader/src/container.ts

```diff
@@ -151,6 +151,34 @@ export enum ConnectionState {
     Connected,
 }
 
+// It specifies the summary taken by the serialize api on detached container.
+export interface IDetachedContainerSnapshot {
+    summary: ISummaryTree | ISnapshotTree,
+    // In format version >= 0.1, we would always have summary. However due to back-compat
+    // we would always have ISnapshotTree for version 0.0.
+    formatVersion: string,
```

Comment:
Want to ask, can we avoid back compat here as I don't think anybody uses the serialize and rehydrate flow yet.

Reply:
That is correct, it's not used today. Though at some point we will bring it back